deny.toml

```diff
@@ -44,7 +44,6 @@ exceptions = [
 
 [bans]
 # Deny multiple versions or wildcard dependencies.
-multiple-versions = "deny"
```

Comment:
Don't change this.

If you did, you should also remove the comment that no longer applies. But don't.

Reply:
We don't have a Cargo.lock file for the settings SDK since it contains only library crates. This makes it impossible to predict `cargo-deny` errors on licensing and multiple versions, but the multiple versions are definitely the "flappiest," and don't generally have a route to resolving them.

We could introduce a lockfile to control when the CI will break. The benefit would be that we could see if introducing a new dependency introduces undue crate duplication downstream without randomly breaking CI. I advocated for disabling the `multiple-versions` check under the assumption that maintaining the lockfile wasn't worth the benefits.

Reply:
> We could introduce a lockfile to control when the CI will break. The benefit would be that we could see if introducing a new dependency introduces undue crate duplication downstream without randomly breaking CI. I advocated for disabling the multiple-versions check under the assumption that maintaining the lockfile wasn't worth the benefits.

If we need to do this, that's fine, but it should be in its own PR and not an unrelated one.

Reply:
Removed the commit from this PR. The PR that fixes this is PR #48. I will rebase to fix build errors.